src/server/server.ts

```diff
@@ -110,16 +112,20 @@ export default class Server implements Hub.RouteBuilder {
 
     this.route("/actions/:actionId/execute", async (req, res) => {
       const request = Hub.ActionRequest.fromRequest(req)
-      const action = await Hub.findAction(req.params.actionId, {lookerVersion: request.lookerVersion})
-      const actionResponse = await action.validateAndExecute(request)
-
-      // Some versions of Looker do not look at the "success" value in the response
-      // if the action returns a 200 status code, even though the Action API specs otherwise.
-      // So we force a non-200 status code as a workaround.
-      if (!actionResponse.success) {
-        res.status(400)
+      const action = await Hub.findAction(req.params.actionId, { lookerVersion: request.lookerVersion })
+      if (action.expensive && action.usesStreaming) {
```

Comment:
let's not bother to check both things here – if we say its expensive we'll fork it